components/Board.tsx

```diff
@@ -5,42 +5,42 @@ export default function Board() {
   return (
     <div className="board">
       {" "}
-      <div className="row" id="gameRow">
+      <div id="gameRow">
```

Comment:
Is there a reason why we are still using id's instead of classNames here now that we have fixed the conflict? My understanding was that there is ideally supposed to be only one of each id at a time and classes were ideally used for styling.